doc/radosgw/group.rst

```diff
@@ -0,0 +1,100 @@
+===============
+Groups
+===============
+
```

Comment:
I think it's a bit late to be putting a new feature into Luminous, though that's up to @mattbenjamin .